sunpy/map/map_factory.py

```diff
@@ -160,7 +160,7 @@ def _read_file(self, fname, **kwargs):
         try:
             pairs = read_file(os.fspath(fname), **kwargs)
         except Exception as e:
-            msg = f"Failed to read {fname}."
+            msg = f"Failed to read {fname}"
```

Comment:
This annoys me when filepaths have an extra "." at the end and I copy it and then try to open it or less the file in the terminal.

Reply:
Ooh yes 👍👍 👍 👍 👍 👍 👍 👍 I noticed this earlier today